pkg/client/resource_quotas_test.go

```diff
@@ -201,8 +201,8 @@ func TestInvalidResourceQuotaUpdate(t *testing.T) {
 		Response: Response{StatusCode: 200, Body: resourceQuota},
 	}
 	_, err := c.Setup().ResourceQuotas(ns).Update(resourceQuota)
-	if err == nil {
-		t.Errorf("Expected an error due to missing ResourceVersion")
+	if err != nil {
+		t.Errorf("no error from client expected: %v", err)
```

Comment:
Not sure if we need this test (`TestInvalidResourceQuotaUpdate`) anymore now that it's not invalid.